docs-2.0/3.ngql-guide/3.data-types/1.numeric.md

```diff
@@ -0,0 +1,13 @@
+# Numeric types
+
+## Integer
+
+An integer is declared with keyword `int` , which is 64-bit *signed*. The supported range is [-9223372036854775808, 9223372036854775807]. Integer constants support multiple formats:
+
+1. Decimal, for example `123456`
+2. Hexadecimal, for example `0xdeadbeaf`
+3. Octal, for example `01234567`
+
+## Double floating point
```

Comment:
I think it's indeed double-precision floating point.

Reply:
I think it's ok here because there is an explanation for doulbe